data/migrations/20200619105012_create-user-table.js

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
You should rename the tables to use lowercase only - ei. user, user_playlists, etc.